designs/tripleo/osc-tripleo.md

```diff
@@ -105,6 +119,8 @@ outputs:
                   - {get_param: OSCVolumeSrc}:/opt/vmidc/bin/data/
             environment:
               - OSC_ENV_VAR={get_param: OSCEnvVar}
+              - TripleO::Services::OSC::tls_truststore_file: {get_param:OSCTrustStoreFile}
```

Comment:
i would add information here about passing in the keystore and truststore passwords as environment variables.

Reply:
Yes this would be addressed in yet another user story.